docs/content.zh/docs/core-concept/transform.md

```diff
@@ -131,18 +131,18 @@ Flink CDC uses [Calcite](https://calcite.apache.org/) to parse expressions and [
-| CURRENT_TIMESTAMP | currentTimestamp() | Returns the current SQL timestamp in the local time zone, the return type is TIMESTAMP_LTZ(3). |
-| NOW() | now() | Returns the current SQL timestamp in the local time zone, this is a synonym of CURRENT_TIMESTAMP. |
-| DATE_FORMAT(timestamp, string) | dateFormat(timestamp, string) | Converts timestamp to a value of string in the format specified by the date format string. The format string is compatible with Java's SimpleDateFormat. |
-| TIMESTAMPDIFF(timepointunit, timepoint1, timepoint2) | timestampDiff(timepointunit, timepoint1, timepoint2) | Returns the (signed) number of timepointunit between timepoint1 and timepoint2. The unit for the interval is given by the first argument, which should be one of the following values: SECOND, MINUTE, HOUR, DAY, MONTH, or YEAR. |
-| TO_DATE(string1[, string2]) | toDate(string1[, string2]) | Converts a date string string1 with format string2 (by default 'yyyy-MM-dd') to a date. |
-| TO_TIMESTAMP(string1[, string2]) | toTimestamp(string1[, string2]) | Converts date time string string1 with format string2 (by default: 'yyyy-MM-dd HH:mm:ss') to a timestamp, without time zone. |
+| Function                                              | Janino Code              | Description                                       |
+|-------------------------------------------------------| ------------------------ | ------------------------------------------------- |
+| LOCALTIME                                             | localtime() | Returns the current SQL time in the local time zone, the return type is TIME(0). |
+| LOCALTIMESTAMP                                        | localtimestamp() | Returns the current SQL timestamp in local time zone, the return type is TIMESTAMP(3). |
+| CURRENT_TIME                                          | currentTime() | Returns the current SQL time in the local time zone, this is a synonym of LOCAL_TIME. |
+| CURRENT_DATE                                          | currentDate() | Returns the current SQL date in the local time zone. |
+| CURRENT_TIMESTAMP                                     | currentTimestamp() | Returns the current SQL timestamp in the local time zone, the return type is TIMESTAMP_LTZ(3). |
+| NOW()                                                 | now() | Returns the current SQL timestamp in the local time zone, this is a synonym of CURRENT_TIMESTAMP. |
+| DATE_FORMAT(timestamp, string)                        | dateFormat(timestamp, string) | Converts timestamp to a value of string in the format specified by the date format string. The format string is compatible with Java's SimpleDateFormat. |
+| TIMESTAMP_DIFF(timepointunit, timepoint1, timepoint2) | timestampDiff(timepointunit, timepoint1, timepoint2) | Returns the (signed) number of timepointunit between timepoint1 and timepoint2. The unit for the interval is given by the first argument, which should be one of the following values: SECOND, MINUTE, HOUR, DAY, MONTH, or YEAR. |
```

Comment:
Maybe we can leave changes of `TIMESTAMP_DIFF` in FLINK-36647

Reply:
The current description in the document is incorrect. This modification is to correct the document.

Reply:
I see, this change makes sense.